docs/TektonResult.md

```diff
@@ -80,7 +80,7 @@ kubectl apply -f pvc.yaml
   kubectl get tektonresults.operator.tekton.dev
   ```
 
-## Properties
+## Spec
 The TektonResult CR is like below:
```

Comment:
Now with this PR changes TektonResult spec will just have 
 ```
loki_stack_name: #optional
loki_stack_namespace: #optional
```
can you update the doc with current spec state instead of entire thing :thinking: 

So basically my point is can we have a clarification in the doc on how `TektonResult` CR looks like what are those 2 format old and new way